src/main/java/com/catapi/model/Breed.java

```diff
@@ -0,0 +1,65 @@
+package com.catapi.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "breed")
+public class Breed {
+
+    @Id
+    @Column(name = "id")
+    private long id;
+
+    @Column(name = "outer_breed_id")
+    private String outerBreedId;
+
+    @Column(name = "outer_name")
+    private String breedName;
+
+    @Column(name = "description")
+    private String description;
+
+    public Breed() {}
+
+    public Breed(long id, String outerBreedId, String breedName, String description) {
+        this.id = id;
```

Comment:
remove id field from the constructor. It will be generated by the database itself

Reply:
Removed